packages/graphback-datasync/src/scheduler.ts

```diff
@@ -0,0 +1,76 @@
+import * as Agenda from "agenda";
+import { Db, MongoClient } from "mongodb";
+
+export interface PruneConfiguration {
+  agendaConfig?: Agenda.AgendaConfiguration
+  pruneInterval?: string | number
+}
+
+export interface TTLMap {
```

Comment:
How is this constructed? 

Via something like a `@ttl` annotation in the model? Or an argument for `@datansync` annotation?

Reply:
It is constructed in code, passed as argument to createDataSyncMongoDbProvider
But what you said sounds really cool! I think it would be nice if it was an argument to datasync :smiley: 
wdyt @wtrocki

Reply:
> It is constructed in code, passed as argument to createDataSyncMongoDbProvider

I saw the part where it is passed as argument in `createDataSyncMongoDbProvider` but not the part where the ma is constructed.

Reply:
Sorry, I see what you mean, we are not constructing it,  I am thinking about constructing it from something like: `@datasync(TTL: 3600)` should be good, what do you think?

Reply:
Yep. see my comment above. That will be exactly the target, however TTL will be specific to some strategies so we might need to figure that out later.

Reply:
+1 on `@datasync(ttl: 3600)`. It looks like this won't work without the `datasync` annotation anyway, so it should not be as a separate annotation.